sigs.yaml

```diff
@@ -46,10 +49,10 @@ sigs:
     url: meet.google.com/zfj-woub-xwp
   contact:
     slack: sig-operations
+    mailing_list: community@lists.operate-first.cloud
   subprojects:
   - name: cluster-ops
-    description: Day to day operations of all the clusters that are part of the Operate
-      First Cloud Offering.
+      description: Day to day operations of all the clusters that are part of the Operate First Cloud Offering.
```

Comment:
please dont touch this line, now it's indented wrong

Reply:
Fixed, thanks for the catch and the etiquette lesson.